src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4610,11 +4610,12 @@ static void Test(string[] args)
     @"using System;
 class TestClass
 {
+    private const string {|Rename:Value|} = $""Text{{s}}"";
```

Comment:
this case is a bit odd.  i can't see a definition for 's'.  So it seems like the original code was in error.  That said, for code in error, i don't really care what we do.  So we can either make this a constant or not.